scenario/context/CometContext.ts

```diff
@@ -138,18 +130,13 @@ export class CometContext {
 
     // Set new supply caps in Configurator and do a deployAndUpgradeTo
     if (shouldUpgrade) {
-      const [targets, values, signatures, calldata] = [[], [], [], []];
-      for (const asset in newSupplyCaps) {
-        targets.push(configurator.address);
-        values.push(0);
-        signatures.push('updateAssetSupplyCap(address,uint128)');
-        calldata.push(utils.defaultAbiCoder.encode(['address', 'uint128'], [asset, newSupplyCaps[asset]]))
+      const admin = await this.world.impersonateAddress(await comet.governor(), 10n**18n);
+      const cometAdmin = (await this.getCometAdmin()).connect(admin);
```

Comment:
not necessary for this PR, but we should rename this function to `getCometProxyAdmin`; it's confusing that we use the term admin to refer to both signers and contracts

Reply:
oh true, we should just call the actor `governor` then

Reply:
Fixed